Gifski/Constants.swift

```diff
@@ -1,12 +1,27 @@
 import SwiftUI
 import CoreTransferable
 import AVFoundation
+import Defaults
 
 enum Constants {
 	static let allowedFrameRate = 3.0...50.0
 	static let loopCountRange = 0...100
 }
 
+
+
+extension CGRect: Defaults.Serializable {
+	static let initialCropRect: CGRect = .init(x: 0, y: 0, width: 1, height: 1)
+}
+/// Codable Rect for Cropping
+/// The coordinates are as follows:
+/// - The origin in the upper left hand corner
+///  - x is in [0,1]
+///  - y is in [0,1]
+///  It's just like a texCoords in a UV texture
+///  in common graphics subroutines.
+typealias CropRect = CGRect
+
```

Comment:
This is moot now.

Reply:
```suggestion
/// Rect for Cropping
/// The coordinates are as follows:
/// - The origin in the upper left hand corner
///  - x is in [0,1]
///  - y is in [0,1]
///  It's just like a texCoords in a UV texture
///  in common graphics subroutines.
typealias CropRect = CGRect

```
Good catch. We could keep the typealias just to denote that the bounds are expected to be between 0-1

Reply:
Sounds like you may want this type instead then: https://developer.apple.com/documentation/swiftui/unitpoint#